src/software/jetson_nano/display/screens/screen.py

```diff
@@ -0,0 +1,126 @@
+from PIL import ImageFont
+import subprocess
+
+"""
+All new screens will inherit from this class. This class will handles editing variables and maintaining
+the current action the cursor is hovering.
+"""
+
+
+class Screen:
+    def __init__(self, lcd_display, status_codes, actions, action_map, draw_screen):
+        self.lcd_display = lcd_display
+        self.font_size = 12
+        self.font = ImageFont.truetype(
+            "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", self.font_size
+        )
+        self.big_font_size = 22
+        self.big_font = ImageFont.truetype(
+            "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", self.big_font_size
+        )
+        self.status_codes = status_codes
+        self.draw_screen = draw_screen
+
+        # Screen actions
+        self.actions = actions
+        self.action_map = action_map
+
+        # Maintain current action
+        self.len = len(self.actions)
+        self.curr_action = 0
+
+        self.edit_mode = False
+        self.param = None
+        self.setting = None
+        self.delta = None
+
+    def draw_header(self):
+        """ Draw the display header """
+        self.lcd_display.draw.rectangle(
+            (0, 0, self.lcd_display.width, self.font_size + 6),
+            outline="#000000",
+            fill="#ffffff",
+        )
+
+        # Get IP address
+        try:
+            cmd = "hostname -I | cut -d' ' -f1"
+            IP = "  IP: " + subprocess.check_output(cmd, shell=True).decode("utf-8")
```

Comment:
ip and signal strength seem like nice things to pull into util functions

Reply:
added a utils.py file with the two functions